### PR TITLE
PlayRippleSound for 939 and UsableRipple Enum

### PR DESCRIPTION
**Pull request by xNexus-ACS**

### CODE REVIEW

**Review by Michal78900 — APPROVED** [review]

File: Exiled.API/Features/Roles/Scp939Role.cs [hunk 1]
```diff
@@ -206,5 +210,29 @@ public void ClearRecordings()
...
+        /// </summary>
+        /// <param name="ripple">The RippleType to play to 939.</param>
+        /// <param name="position">The Sync Position to play.</param>
+        /// <param name="playerToSend">The Player to send the Ripple Sound.</param>
+        public void PlayRippleSound(UsableRippleType ripple, Vector3 position, Player playerToSend)
+        {
+            switch (ripple)
+            {
+                case UsableRippleType.Footstep:
+                    SubroutineModule.TryGetSubroutine(out FootstepRippleTrigger footstepRippleTrigger);
```
Comment (louis1706) [inline Exiled.API/Features/Roles/Scp939Role.cs:225]: put subroutine in memory in the cttor

File: Exiled.API/Features/Roles/Scp939Role.cs [hunk 2]
```diff
@@ -206,5 +210,29 @@ public void ClearRecordings()
...
+                    footstepRippleTrigger._syncPos = new RelativePosition(position);
+                    footstepRippleTrigger.ServerSendRpc(playerToSend.ReferenceHub);
+                    break;
+                case UsableRippleType.FireArm:
+                    SubroutineModule.TryGetSubroutine(out FirearmRippleTrigger firearmRippleTrigger);
```
Comment (louis1706) [inline Exiled.API/Features/Roles/Scp939Role.cs:230]: Same

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Roles/Scp939Role.cs (hunk 1 above)
Comment (xNexus-ACS) [reply Exiled.API/Features/Roles/Scp939Role.cs]: Done

File: Exiled.API/Features/Roles/Scp939Role.cs (hunk 2 above)
Comment (xNexus-ACS) [reply Exiled.API/Features/Roles/Scp939Role.cs]: Done

**Review by louis1706 — APPROVED** [review]

**Comment by louis1706** [comment]
@xNexus-ACS  is it tested ?

**Comment by xNexus-ACS** [comment]
> @xNexus-ACS is it tested ?

yes

**Pull request merged**